### DISCUSSION TITLE
article/j6ixx1di/

### DISCUSSION THREAD

**Discussion by giscus** [yxzlwz/blog#10] [Announcements]
# article/j6ixx1di/

前两天，微信公众号【何加盐】发布了文章《中文互联网正在加速崩塌》，一夜之间在中文互联网刷屏，又一夜之间消失。 我是在这篇文章被删除后才从Twitter上了解到它，从Archive Today (https://archive.md/HoGem)上阅读了被存档的原文后，觉得作者写得真的深得我心。原文我已经转载在自己的博客上，大家可以前往阅读： 在开始前先...

**Comment by yxzlwz** [comment]
本文微信公众号同步发布链接：<https://mp.weixin.qq.com/s/UNdd2rVtPK-Lg1gumL-NyA>；在知乎发布的文章未能通过审核。
